File: nec_code_mumbai_sep22/nec code/spring_boot/SpringApp11/src/main/java/com/abc/UserInfo.java
package com.abc;

import lombok.Data;

@Data
public class UserInfo  implements User{

	private int userId;
	private String userName;
	private String userLocation;
	
	
	
	@Override
	public void getUserInfo() {
	
		System.out.println("\nUser Information");
		System.out.println("user id "+getUserId());
		System.out.println("user name "+getUserName());
		System.out.println("user location "+getUserLocation());
		
	}

}

File: react/reactapp1/src/user/userinfo.js
function UserInfo() {
    return (
        <div>
            <h2>User Information</h2>
            <div>
                id is {userinfo.id}
            </div>
            <div>
                name is {userinfo.name}
            </div>
            <div>
                age is {userinfo.age}
            </div>

        </div>
    )
}
export default UserInfo;
const userinfo = {
    "id": 10001, "name": "suresh kumar", "age": 30, "gender": "male"
}